library/azure_rm_virtualmachine.py

```diff
@@ -790,7 +789,7 @@
 
 def extract_names_from_blob_uri(blob_uri, storage_suffix):
     # HACK: ditch this once python SDK supports get by URI
-    m = re.match(r'^https://(?P<accountname>[^.]+)\.blob\.{0}/'
+    m = re.match(r'^https?://(?P<accountname>[^.]+)\.blob\.{0}/'
```

Comment:
It seems this change is not included in the two PRs?

Reply:
It's an history PR, got merged recently. So actually three PRs got merged.

Reply:
And that one is a community PR...